lib/src/native/realm_core.dart

```diff
@@ -70,7 +70,7 @@ class _RealmCore {
     return _instance ??= _RealmCore._();
   }
 
-  String get libraryVersion => _realmLib.realm_get_library_version().cast<Utf8>().toDartString();
+  String get libraryVersion => _realmLib.realm_get_library_version().toDartString()!;
```

Comment:
I suggest we keep the `cast<Utf8>`. 
This change enables toDartString on every `Pointer<Int8>` which is not a string every time. Being more explicit here is intentional.

Reply:
In any case, changed it to be an extension on Pointer<Utf8> - any time either approach would have saved is probably not worth debating this further.